frontendproject.md

```diff
@@ -0,0 +1,17 @@
+# Frontend Developer Challenge
+
+Choose a MVVM/Redux framework of your choice. While we encourage using React, Vue or Angular, please use whatever framework and language you are most comfortable with. Our application frontend is written in in Backbone and React.
```

Comment:
I think we should make this more explicit. I'd rather have them choose one of those three, or another framework we feel qualifies as good experience. @th3fallen?

Reply:
One of the three suggestions will hopefully be there most used anyway. I'm not sure jailing them to those solves anything but we should imply to those three are very strongly and suggested. But really have no good argument for either side.